commerce-service/src/main/resources/content/Language.properties

```diff
@@ -5,14 +5,21 @@ category.users=Users
 display-discount-levels=Display Discount Levels
 guest-cart-item-max-allowed=Guest Cart Item Max Allowed
 guest-cart-max-allowed=Guest Cart Max Allowed
+order-approved=Order Approved
+order-awaiting-shipment=Order Awaiting for Shipment
```

Comment:
`Order Awaiting Shipment` (or `Order Waiting for Shipment`), not `Order Awaiting for Shipment`.